class-wxr-importer.php

```diff
@@ -1568,7 +1574,22 @@ protected function process_author( $data, $meta ) {
 			$user_id
 		) );
 
-		// TODO: Implement meta handling once WXR includes it
+		// Update user meta.
+		foreach ( $meta as $meta_item ) {
+			$key = $meta_item['key'] ?? false;
+			$value = $meta_item['value'] ?? false;
+
+			// export gets meta straight from the DB so could have a serialized string
+			if ( ! $value ) {
+				$value = maybe_unserialize( $value );
```

Comment:
**question:** Isn't `$meta_item['value']` always a `string` even when it is serialized PHP? Currently the `maybe_unserialize` appears to be called only when the `$value` is actually `false`, `null`,  `''`, `0`.